getting-started/installation-windows-msys2.rst

```diff
@@ -0,0 +1,290 @@
+    epicsEnvSet("TOP","C:/msys64/home/'user'/testioc")
+    epicsEnvSet("EPICS_BASE","C:/msys64/home/'user'/base-7.0.4.1")
+
+**Note:** You can use Linux style forward slash characters in path specifications inside this file or double backslashes (``\\``).
+
+At this point, you can run the IOC from either an MSYS2 Bash shell or from a Windows command prompt, by changing into the IOC directory and running the test.exe binary with your startup command script as parameter.
+
+In the Windows ``command prompt``::
+
+    >cd C:\msys64\home\'user'\testioc\iocBoot\ioctest    
+    >..\..\bin\windows-x64-mingw\test st.cmd
+
+In the MSYS2 shell::
+
+    $ cd ~/testioc/iocBoot/ioctest    
+    $ ../../bin/windows-x64-mingw/test st.cmd
+
+
+In both cases, the IOC should start like this::
+
+    Starting iocInit
+    iocRun: All initialization complete
+    #!../../bin/windows-x64-mingw/test
+    < envPaths
+    epicsEnvSet("IOC","ioctest")
+    epicsEnvSet("TOP","C:/msys64/home/'user'/testioc")
+    epicsEnvSet("EPICS_BASE","C:/msys64/home/'user'/base-R7.0.4.1")
+    cd "C:/msys64/home/'user'/testioc"
+    ## Register all support components
+    dbLoadDatabase "dbd/test.dbd"
+    test_registerRecordDeviceDriver pdbbase
+    Warning: IOC is booting with TOP = "C:/msys64/home/'user'/testioc"
+              but was built with TOP = "/home/'user'/testioc"
+    ## Load record instances
+    dbLoadRecords("db/test.db","user='user'")
+    cd "C:/msys64/home/'user'/testioc/iocBoot/ioctest"
+    iocInit
+    ############################################################################
+    ## EPICS R7.0.4.1
+    ## Rev. 2020-10-21T11:57+0200
+    ############################################################################
+    ## Start any sequence programs
+    #seq sncxxx,"user='user'"
+    epics>
+
+Check if the database ``test.db`` you created is loaded correctly::
+
+    epics> dbl
+    test:pv1
+    test:pv2
+    test:add
+
+As you can see 3 process variable is loaded and available. Keep this terminal open and running. Test this process variable using another terminals.
+
+Open another shell for monitoring ``test:add``::
+
+    $ camonitor test:add
+    test:add                       2020-10-23 13:39:14.795006 100
+
+That terminal will monitor the PV ``test:add`` continuously. If any value change is detected, it will be updated in this terminal. Keep it open to observe the behaviour.
+
+Open a third shell. Using caput, modify the values of  ``test:pv1`` and ``test:pv2`` as we have done in the temperature example above. You will see changes of their sum in the second terminal accordingly.
+
+At this point, you have one IOC ``testioc`` running, which loaded the database ``test.db`` with 3 records. From other processes, you can connect to these records using Channel Access. If you add more process variable in ``test.db``, you will have to ``make`` the `testioc` application again and restart the IOC to load the new version of the database.
```

Comment:
you will not have to ``make`` the `testioc` application again, just restart the IOC to load the new version of the database.

Reply:
You have to re-run make, else the database files (in `...App/Db`) will not get installed (in `TOP/db`) where the IOC loads them from.